docs/java/java-tutorial.md

```diff
@@ -111,19 +112,19 @@ To learn more about editing Java, see [Java Editing](/docs/java/java-editing.md)
 
 ## Running and debugging your program
 
-To run and debug Java code, set a breakpoint, then either press `kb(workbench.action.debug.start)` on your keyboard or use the **Run** > **Start Debugging** menu item. You can also use the **Run|Debug** CodeLens options in the editor. After the code compiles, you can see all your variables and threads in the Run view.
+To run and debug Java code, set a breakpoint, then either press `kb(workbench.action.debug.start)` on your keyboard or use the **Run** > **Start Debugging** menu item. You can also use the **Run|Debug** CodeLens option in the editor. After the code compiles, you can see all your variables and threads in the Run view.
 
 <video autoplay loop muted playsinline controls>
   <source src="/docs/java/java-tutorial/run-debug.mp4" type="video/mp4">
 </video>
 
-The debugger also supports advanced features such as Hot Code replacement and conditional breakpoints.
+The debugger also supports advanced features such as Hot Code Replacement and conditional breakpoints.
```

Comment:
Looks like the feature is called "Hot Code Replace". 
@nickzhums Should we be using that term wherever the docs currently say "Hot Code replacement"?

Reply:
I'll update to be "Hot Code Replace" to match the feature name

Reply:
For future reference, that was done in commit 3a2469482075d54750f7856d3e2c2cde12f78e10.